gqcpy/include/interfaces.hpp

```diff
@@ -472,7 +472,21 @@ void bindSimpleTransformationInterface(Class& py_class) {
 
         .def("matrix",
              &Type::matrix,
-             "Return the transformation matrix that collects the expansion coefficients of the new basis (vectors) in the old basis as columns.");
+             "Return the transformation matrix that collects the expansion coefficients of the new basis (vectors) in the old basis as columns.")
+
+        .def("inverse",
+             &Type::inverse,
+             "Return the inverse transformation of this transformation matrix.")
+
+        .def(
+            "isUnitary",
+            [](const Type& T, const double threshold = 1.0e-12) {
+                return T.isUnitary(threshold);
+            },
```

Comment:
```suggestion
            &Type::isUnitary,
            py::arg("threshold") = 1.0e-12,
```

I think this can be written shorter.